src/frequenz/client/common/microgrid/microgrid.py

```diff
@@ -0,0 +1,59 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Module to define the microgrid used with the common client."""
+
+from __future__ import annotations  # required for constructor type hinting
+
+import logging
+from dataclasses import dataclass
+from typing import List
+
+# pylint: disable=import-error, no-name-in-module
```

Comment:
I think `no-name-in-module` should be enough here. Also we want to enable it again after the import with

`# pylint: enable=no-name-in-module`

the CI error is related to mypy, not sure whats up there